docs/blog/2019-01-30-flamingo-case-study/index.md

```diff
@@ -1,7 +1,5 @@
 ---
-title:
-  "Flamingo by Harry’s chooses Gatsby for critical ecommerce launch, 5X faster
-  page loads"
+title: "Flamingo, women’s body care brand from Harry’s, chooses Gatsby for critical ecommerce launch, 5X faster page loads"
```

Comment:
This feels kinda ungainly. What's the goal of the change? The previous was more direct and snappy.